python/manylinux1/build_arrow.sh

```diff
@@ -48,40 +45,43 @@ export PYARROW_CMAKE_OPTIONS='-DTHRIFT_HOME=/usr'
 # Ensure the target directory exists
 mkdir -p /io/dist
 
-for PYTHON in ${PYTHON_VERSIONS}; do
-    PYTHON_INTERPRETER="$(cpython_path $PYTHON)/bin/python"
-    PIP="$(cpython_path $PYTHON)/bin/pip"
-    PIPI_IO="$PIP install -f $MANYLINUX_URL"
-    PATH="$PATH:$(cpython_path $PYTHON)"
+for PYTHON_TUPLE in ${PYTHON_VERSIONS}; do
+    IFS=","
+    set -- $PYTHON_TUPLE;
+    PYTHON=$1
+    U_WIDTH=$2
+    PYTHON_INTERPRETER="$(cpython_path $PYTHON ${U_WIDTH})/bin/python"
```

Comment:
Looks like the result of `$(cpython_path $PYTHON ${U_WIDTH})` could be stored in a dedicated variable?